src/mcp/util/detect.ts

```diff
@@ -0,0 +1,27 @@
+import { Config } from "../../config";
+
+export type WorkspacePlatform = "web" | "android" | "ios" | "flutter" | "react-native" | "unity";
+
+export function detectWorkspacePlatform(config: Config): WorkspacePlatform | null {
```

Comment:
This is super similar to getPlatformFromFolder in src/dataconnect/appFinder.ts. Can we pick one implementation and stick to that?